spec/controllers/slides_controller_spec.rb

```diff
@@ -120,7 +120,7 @@
       slide_id = Slide.where("user_id = #{default_user.id}").first.id
       get :edit, params: { id: slide_id }
       expect(response.status).to eq(200)
-      expect(response).to render_template 'slides/aws/edit'
+      expect(response).to render_template "slides/#{CloudConfig::service_name}/edit"
```

Comment:
__[RuboCop]__

Style/ColonMethodCall: Do not use `::` for method calls.


<!-- Comment by [SideCI](https://sideci.com) -->